### ISSUE TITLE
on windows even `wanna --help` takes several seconds

### ISSUE THREAD

**Issue by racinmat** [avast/wanna-ml#116]
### What happened?

wanna-ml is a bit slow even for showing help, investigate

### wanna.yaml file

```yaml
-
```

### wanna-ml CLI command

```shell
-
```

**Comment by jsilva** [comment]
fixed in 0.3.1 release

**Comment by racinmat** [comment]
perfect!
This was case in 0.2.7:
```
(wanna-heur-eml) PS C:\Projects\others\wanna-heur-eml> Measure-Command { wanna --help }

Days              : 0                                  
Hours             : 0                                  
Minutes           : 0                                  
Seconds           : 7                                  
Milliseconds      : 420                                
Ticks             : 74201182                           
TotalDays         : 8.58809976851852E-05               
TotalHours        : 0.00206114394444444                
TotalMinutes      : 0.123668636666667                  
TotalSeconds      : 7.4201182                          
TotalMilliseconds : 7420.1182                          
```
and this is with 0.3.1
```
PS C:\Projects\others\wanna-heur-eml> Measure-Command { wanna --help }                                     

Days              : 0                   
Hours             : 0                   
Minutes           : 0                   
Seconds           : 1                   
Milliseconds      : 913                 
Ticks             : 19138872            
TotalDays         : 2.21514722222222E-05
TotalHours        : 0.000531635333333333
TotalMinutes      : 0.03189812          
TotalSeconds      : 1.9138872           
TotalMilliseconds : 1913.8872           
```
so the speedup is great!
